### ISSUE TITLE
flickering test: tom_adds_reviewer_Iggy

### ISSUE THREAD

**Issue by hennevogel** [openSUSE/open-build-service#3001]
This test is flickering:

```
test_tom_adds_reviewer_Iggy                                     FAIL (15.55s)
Minitest::Assertion:         expected to find text "Request 4 (declined)" in "Watchlist Open Build Service 
> Request 4 adrian (3 tasks) | Home Project | Logout Overview Request 4 (review) test home project 
shortening Add a review Created by Iggy over 4 years ago In state review Open review for tom Open 
review for test_group_b Open review for test_group_b Submit BranchPack Submit package 
home:Iggy:branches:kde4 / BranchPack (revision 1) to package Apache / BranchPack [+] Added 
myfile Build Results Rpmlint Results The project this package belongs to currently has no build 
targets defined. Review for test_group Request History Iggy created request over 4 years ago test 
home project shortening Iggy Request got reviewed over 4 years ago tom Request got a new review 
request now tom Request got a new review request now tom Request got a new review request now 
tom Request got a new review request now Iggy Review got accepted now Ok for the project Iggy 
Review got accepted now Ok for the package Iggy Review got accepted now Ok for the package Iggy 
Review got accepted now And ok for me Comments for request 4 (3) Admin wrote over 3 years ago 
This request rules! I am making a comment on a request. Reply tom wrote over 3 years ago And I like 
that comment. Reply adrian wrote over 3 years ago While I don't! Reply Delete adrian: Your Profile 
Home Project Logout Locations Projects Search Status Monitor Help Open Build Service OBS 
Manuals openSUSEs OBS Portal Reporting a Bug Contact Mailing List Forums Chat (IRC) Twitter 
Open Build Service (OBS) is an openSUSE project.".
        test/functional/webui/request_controller_test.rb:244:in `test_tom_adds_reviewer_Iggy'
        test/test_helper.rb:126:in `block in __run'
        test/test_helper.rb:126:in `map'
        test/test_helper.rb:126:in `__run'
```

**Comment by bgeuken** [comment]
Maybe, hopefully fixed by the last commit of #3007. TBH, I wasn't able to reproduce this one, but ran into a few other(see the other 2 commits).

**Comment by bgeuken** [comment]
That's one of the tests that got dropped by now
